src/components/cart.lineitem.jsx

```diff
@@ -30,6 +30,9 @@ import {
 import imgPlaceholder from '../images/img-placeholder.png';
 import cortexFetch from '../utils/Cortex';
 import './cart.lineitem.less';
+import epConfig from '../ep.config.json';
+
+const configurationPrefix = epConfig.cartItemModifier.prefix;
```

Comment:
Not necessary as you can reuse the existing config and pull what you need later in your block with:
`Config.cartItemModifier.prefix`